docs/deploy/howto-deploy-modules-portal.md

```diff
@@ -29,7 +29,8 @@ To deploy all required modules to the Gateway using the Azure Portal...
    As *create options* use the following JSON:
 
    ```json
-   {"NetworkingConfig":{"EndpointsConfig":{"host":{}}},"HostConfig":{"NetworkMode":"host","CapAdd":["NET_ADMIN"]}}
```

Comment:
@dacolgit, can you please also add those values to the deployment manifest in `docs/modules/publisher.md` ? It would also be great if you can add a few lines there explaining the values, so people understand why they should use `CapDrop` in production setups ?

Reply:
done